ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/ConnectorTool.java

```diff
@@ -144,5 +144,10 @@ public String getDefaultType() {
         public String getDefaultVersion() {
             return null;
         }
+
+        @Override
+        public List<String> getAllModelKeys() {
+            return List.of();
+        }
```

Comment:
ConnectorTool is using connector_id instead of model_id.

Reply:
Yes. So we keep it empty.